cypress/e2e/digital-lpa.cy.js

```diff
@@ -42,6 +42,20 @@ describe("View a digital LPA", () => {
               town: "London",
             },
           },
+          linkedDigitalLpas: [
+            {
+              uId: "M-DIGI-LPA3-3334",
+              caseSubtype: "personal-welfare",
+              Status: "Processing",
+              createdDate: "01/11/2023",
+            },
+            {
+              uId: "M-DIGI-LPA3-3335",
+              caseSubtype: "personal-welfare",
+              Status: "Registered",
```

Comment:
`status` should be lower case. I guess the Go json tags must be case insensitive, which is news to me
```suggestion
              status: "Processing",
              createdDate: "01/11/2023",
            },
            {
              uId: "M-DIGI-LPA3-3335",
              caseSubtype: "personal-welfare",
              status: "Registered",
```

Reply:
Good eye 👍